data/en/queryevery.json

```diff
@@ -5,20 +5,62 @@
 	"member": "query.every(closure [, parallel] [, maxThreads])",
 	"returns": "boolean",
 	"related": [],
-	"description": "",
+	"description": "Executes a callback/closure against every row in a query and returns true if the callback/closure returned true for every row.",
 	"params": [
-		{"name": "query", "description": "", "required": true, "default": "", "type": "query", "values": []},
-		{"name": "closure", "description": "", "required": true, "default": "", "type": "function", "values": []},
-		{"name": "parallel", "description": "", "required": false, "default": "", "type": "boolean", "values": []},
-		{"name": "maxThreads", "description": "", "required": false, "default": "20", "type": "numeric", "values": []}
+		{
+			"name": "query",
+			"description": "",
+			"required": true,
+			"default": "",
+			"type": "query",
+			"values": []
+		},
+		{
+			"name": "closure",
```

Comment:
You can use `callback_params` here like so: https://github.com/foundeo/cfdocs/blob/3f33f8d8aacec842205e93d81242cac4d65eac66/data/en/listmap.json#L10

Reply:
I'll merge this PR, @atuttle if you have a chance please add the `callback_params` in another PR.

Reply:
I can do that... I don't see the point though. How is that data used? The list of callback params displayed on https://cfdocs.org/listmap is just from the description attribute of the function param.

Reply:
You're right. This is not as handy as possible yet. But maybe this can be improved.

Reply:
Having the data is handy in itself. The web site is just one potential consumer for the data, and it can be modified over time to utilize it better if it's available. Another use for the data is in editors to facilitate CFML development.